frontend/js/Pages/Error.vue

```diff
@@ -37,6 +37,8 @@
 </template>
 
 <script setup>
+import route from "ziggy";
+
 const props = defineProps({
   status: Number,
 });
```

Comment:
Move setup section above of the template element - setup section is above of the template in components.